Makefile

```
@@ -216,7 +216,7 @@ lint: .k0sbuild.docker-image.k0s go.sum codegen
 	$(GO_ENV) golangci-lint run --verbose $(GOLANGCI_LINT_FLAGS) $(GO_DIRS)
 
 airgap-images.txt: k0s
```

```suggestion
airgap-images.txt: k0s .k0sbuild.docker-image.k0s
```

If the Docker container is used to execute the binary, it's better to list it as a make prerequisite.

---

Done